jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  testPathIgnorePatterns: [
+    '/frontend/',
+    '/node_modules/',
+    'test/functional',
```

Comment:
Since we're modifying this line..

```suggestion
    '/test/functional/',
```